plugins/git/receive-app

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+APP="$1"; REV="$2"
+
+dokku git-build $APP $REV
```

Comment:
Oh except, derp!! We moved it to commands so the `ps:rebuild`, et al. would work. So nevermind, carry on, nothing to see here.